src/localization/locales/en-US.json

```diff
@@ -714,7 +715,7 @@
         "validation_url_required": "Enter a web address.",
         "validation_url_invalid": "Enter a valid web address.",
         "validation_field_required_array": "$t({{path}}) {{arrayIndex}} is required",
-        "validation_field_max_length": "$t({{path}}) must be fewer than {{max}} characters.",
+        "validation_field_max_length": "$t({{path}}) is too long. Please add a description that is fewer than {{max}} characters.",
```

Comment:
issue: this is too wordy. Please revert this change. This also appliels to fields other than the description.